src/components/internal/DatePicker/Header.tsx

```diff
@@ -18,3 +18,24 @@ export function Header(props: CaptionProps) {
     </div>
   );
 }
+
+// Header with year skip option
+export function YearSkipHeader(props: CaptionProps) {
+  const { displayMonth } = props;
+  const { goToMonth } = useNavigation();
+
+  return (
+    <div css={Css.df.jcsb.aic.mlPx(12).mrPx(12).hPx(32).$}>
+      <div css={Css.df.fdr.jcsb.$ }>
+        <IconButton color={Palette.Gray700} icon="chevronLeft" onClick={() => goToMonth(addMonths(displayMonth, -1))} />
+        <h1 css={Css.base.$}>{format(displayMonth, "MMM")}</h1>
+        <IconButton color={Palette.Gray700} icon="chevronRight" onClick={() => goToMonth(addMonths(displayMonth, 1))} />
+      </div>
+      <div css={Css.df.fdr.jcsb.$ }>
+        <IconButton color={Palette.Gray700} icon="chevronLeft" onClick={() => goToMonth(addYears(displayMonth, -1))} />
+        <h1 css={Css.base.$}>{format(displayMonth, "yyyy")}</h1>
+        <IconButton color={Palette.Gray700} icon="chevronRight" onClick={() => goToMonth(addYears(displayMonth, 1))} />
+      </div>
+    </div>
+  );
+}
```

Comment:
When using dateRangeFilter we have to support undefined. Created this second header to make date selections easier on clear.

![Screen Shot 2022-08-08 at 11 50 48 AM](https://user-images.githubusercontent.com/108819408/183471364-408df1b3-8696-400a-bf7a-b6cef1848c71.png)